### ISSUE TITLE
Gemfile w/ duplicate path gems fails with Bundler > 2.2.7

### ISSUE THREAD

**Issue by internets** [heroku/heroku-buildpack-ruby#1135]
When #1132 was merged our build began to fail due to a bug introduce in Bundler 2.2.8. I submitted a bugfix PR that was just incorporated in the latest 2.2.12 release. I understand that updating bundler in the ruby buildpack is not done lightly, so please let me know if an update is not in the cards :)

If 2.2.12 doesn't pass muster we will happily maintain a monkeypatch with a version check to remind us to remove it when it is updated, so no rush!

Bundler 2.2.12 changelog: https://github.com/rubygems/rubygems/blob/bundler-v2.2.12/bundler/CHANGELOG.md#2212-march-1-2021

**Comment by schneems** [comment]
Working on this in https://github.com/heroku/heroku-buildpack-ruby/pull/1140

I'll likely not release until next week though.

**Comment by schneems** [comment]
Tests are failing on that branch but still work on main:

```
       remote:        Gem::Ext::BuildError: ERROR: Failed to build gem native extension.        
       remote:                
       remote:        current directory:        
       remote:        /tmp/build_392437cc/vendor/bundle/jruby/2.5.0/gems/naether-0.15.7-java        
       remote:        jrake        
       remote:        RUBYARCHDIR=/tmp/build_392437cc/vendor/bundle/jruby/2.5.0/extensions/universal-java-1.8/2.5.0/naether-0.15.7-java        
       remote:        RUBYLIBDIR=/tmp/build_392437cc/vendor/bundle/jruby/2.5.0/extensions/universal-java-1.8/2.5.0/naether-0.15.7-java        
       remote:        sh: 1: jrake: not found        
```

This might be a regression in bundler. I'll have to investigate more.

**Comment by internets** [comment]
Thanks! No rush, glad to know it's in progress.

That test failure does look like a bundler regression in 2.2.12. I suspect it's fixed by https://github.com/rubygems/rubygems/pull/4428

**Comment by internets** [comment]
Closed by #1140 -- thanks!
